plugins/dell-dock/fu-dell-dock-i2c-ec.c

```diff
@@ -436,7 +437,7 @@ fu_dell_dock_ec_get_dock_info (FuDevice *device,
 		return FALSE;
 	}
 
-	fu_device_set_version_lowest (device, self->ec_minimum_version, FWUPD_VERSION_FORMAT_PAIR);
+	fu_device_set_version_lowest (device, self->ec_minimum_version);
```

Comment:
Perfect example of why it makes sense to hardcode to same for all 3 versions, this was a bug waiting to happen.